progressEval.html

```diff
@@ -0,0 +1,458 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<meta http-equiv="X-UA-Compatible">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<title>
+Course Progress Evaluator
+</title>
+<style>
+body {
+	font-family: sans-serif;
+	color: black;
+}
+input[type="text"] {
+   color: black;
+}
+
+</style>
+</head>
+<body>
+		<h2>Course Progress Evaluator</h2>	
+		<form role="form" id="grades" name="grades">	
+		<p> What week is it? </p>
+		<select class="" id="weekselect" onchange="calculate()">
+	      <option id="week" value="0">Week 0</option>
```

Comment:
"Week 0" makes no intuitive sense as an option, I'd recommend looking at [this StackOverflow post](https://stackoverflow.com/questions/5805059/how-do-i-make-a-placeholder-for-a-select-box) for some suggestions to force a selection in a dropdown menu.